corehq/apps/cloudcare/views.py

```diff
@@ -134,7 +134,8 @@ def get(self, request, domain, urlPath):
                 apps = get_cloudcare_apps(domain)
             else:
                 apps = get_brief_apps_in_domain(domain)
-            apps = [get_app_json(app) for app in apps if app and app.application_version == V2]
+            apps = [get_app_json(app) for app in apps if app and (
+                isinstance(app, RemoteApp) or app.application_version == V2)]
```

Comment:
This is because while doing this change I also moved the `application_version` property from the base model shared by `RemoteApp` and `Application` to just `Application`. Consequently, `application_version` will not be available on `RemoteApp`s, and adding `isinstance(app, RemoteApp) or` in there preserves previous behavior.